voluptuous/schema_builder.py

```diff
@@ -165,7 +165,7 @@ def Extra(_) -> None:
 # deprecated object, so we just leave an alias here instead.
 extra = Extra
 
-Schemable = typing.Union[dict, list, type, typing.Callable]
+Schemable = typing.Union[dict, list, str, type, typing.Callable]
```

Comment:
I think you'll need numeric, bool, etc. types in here too - basically any non-class/object type can be used. I haven't kept track of what these are in current Python.